.gitignore

```diff
@@ -4,4 +4,5 @@ install/
 .vscode/
 .cursor/
 .idea/
+.DS_Store
```

Comment:
I feel it's better to ignore this in your system-wide `.gitnigore` instead of here.

Reply:
Can I keep it here, just one line, More portable for Mac users. In case, other developers also use MAC, :)

Reply:
My point is that for people developing on Mac, it makes more sense to include this in their global `gitignore` file. Actually, I'd suggest including all of https://github.com/github/gitignore/blob/main/Global/macOS.gitignore in their global `gitignore`. For this project-wise `gitignore`, I'd prefer keeping it as clean as possible.

Reply:
> My point is that for people developing on Mac, it makes more sense to include this in their global `gitignore` file. Actually, I'd suggest including all of https://github.com/github/gitignore/blob/main/Global/macOS.gitignore in their global `gitignore`. For this project-wise `gitignore`, I'd prefer keeping it as clean as possible.

let take look at this

Reply:
Actually, I think it is okay to add this here. Eventually, someone with a MacOS won't have this global setup and then the file is inadvertently added in some bulk commit. It's a single line in the git diff so easy to overlook.

https://github.com/p4lang/p4c/blob/main/.gitignore#L40 has some categorization that can be useful. 

Probably have to add all the dot dirs for the AI tools soon, too :).

Reply:
Okay, let's keep this then.

Reply:
Done